config-linux.md

```diff
@@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
 
 ```json
     "memory": {
-        "limit": 0,
+        "limit": 536870912,
         "reservation": 0,
```

Comment:
If we were to be pedantic, I'd rename all the fields to include the units. For example, `reservation` becomes `reservationBytes`